# FTP::List

string dir_path
---
string[] files
string[] dirs
bool success
